src/casp/services/README.rst

```diff
@@ -24,6 +28,9 @@ The service must also be authenticated with Google credentials, either by settin
 - Running in a Google environment (e.g. Cloud Run) with the service account attached to the service, in which case the service account is
 used as the identity running the servce.
 
+..
+    Maybe a dumb question, but which services are Cromwell executable tasks?  Also, is that technically a service?
```

Comment:
I _think_ that those call the code from this repo but other than that, there's no explicit callout to Cromwell.  Again, I think

Reply:
Yeah initially the idea was that the codebase consist of deployable and pipeline executable services. My opinion that executable services should go away to somewhere else.

Reply:
As we discussed in the meeting: this we should keep until we factor out the scripts.